data/hai/hai reveal 2 start.txt

```diff
@@ -64,22 +88,44 @@ mission "Hai Rescue: Yeertle Family"
+			`	The whole family cheers and it turns out they were already packed to leave. Duniip explains they move regularly so they're not found. It takes them under two minutes to finish preparations and don their holoprojectors. Your walk back to the ship is largely uneventful until some men with strange scanners detect the Yeertle family. One man throws a small grenade with a green light on top. You recognize it as a low-level EMP device. All five holograms turn off, along with all computers within twenty meters. Fortunately, your ship is outside that range.`
 			`	The six of you run to your ship as quickly as possible. You look back and see the men were not following you; they were running to their own ships.`
 				launch
 	on complete
 		log `Found a Hai family that had been locked up by some crooked merchants meant to be providing transport. A lively bunch, returned them to <destination>.`
 		payment 350000
 		"hai slave prereq" ++
-		dialog `The Yeertle family is ecstatic to be home after their long ordeal and gladly give you <payment>. They promise to look at the background checks of human pilots before hiring next time.`
+		dialog `The Yeertle family is ecstatic to be home after their long ordeal and gladly give you <payment>. They promise to run background checks of human pilots before hiring next time.`
 	npc
 		government "Bounty Hunter that Won't Enter Hai Space"
 		personality plunders disables unconstrained
 		fleet "Small Core Pirates" 1
 
 
+mission "Hai Rescue: Prison RP Hint"
+	landing
+	name "Phantom Distress Calls"
+	description "Some merchants have mentioned picking up a distress call they couldn't locate the source of."
+	source
+		near "Mirfak" 4
+		government "Republic" "Syndicate"
+		attributes "spaceport"
+		not attributes "station"
+	destination Sunracer
+	to offer
+		has "Hai Leaks Response 6: offered"
+		has "event: slightly lost"
+		not "Hai Rescue: Prison Role-Players: offered"
+		not "event: hai-human treaty signed"
+		random < 80
+	on offer
+		conversation
+			`As you come in to land at the spaceport you listen to the usual chatter of captains coming and going. Out here on the edge of things, towards the core, it is a popular pastime for merchants to swap ghost stories. In all likelihood the majority of these stories can be put down to minor failures from cheap, mass-produced Syndicate equipment. Nevertheless, something about one of them stands out this time.`
+			`	It seems a handful of merchants have all been picking up a very faint distress signal in the vicinity of Sunracer, but every time they've gone to look for it, they've either experienced some kind of engine warning, or a large group of pirates has jumped into the system and they've bugged out.`
+			`	It seems like a long-shot but you wonder if perhaps this might be a legitimate call for help, maybe from someone who can't risk broadcasting themselves too aggressively...`
```

Comment:
```suggestion
			`	It seems like a long-shot, but you wonder if perhaps this might be a legitimate call for help, maybe from someone who can't risk broadcasting themselves too aggressively...`
```